### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by ionbasa** [42wim/matterbridge#216]
### Which version of matterbridge are you using?
0.16.1-dev 16eb12b

### If you're having problems with mattermost please specify mattermost version. 
3.10

### Please describe the expected behavior.
Should run like normal. Expected behavior works if I specify an account username and password instead of WebHook.

### Please describe the actual behavior. 
```
INFO[2017-07-11T06:21:19Z] Starting bridge: mattermost.work
INFO[2017-07-11T06:21:19Z] Connecting using webhookurl (for posting) and token  module=mattermost
INFO[2017-07-11T06:21:19Z] mattermost.work: joining off-topic (off-topicmattermost.work)
INFO[2017-07-11T06:21:19Z] Gateway(s) started succesfully. Now relaying messages
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x58 pc=0x5561bccc6870]

goroutine 77 [running]:
github.com/42wim/matterbridge/bridge/mattermost.(*Bmattermost).handleMatterClient(0xc42004e9a0, 0xc4203becc0)
        /go/src/github.com/42wim/matterbridge/bridge/mattermost/mattermost.go:143 +0x40
created by github.com/42wim/matterbridge/bridge/mattermost.(*Bmattermost).handleMatter
        /go/src/github.com/42wim/matterbridge/bridge/mattermost/mattermost.go:134 +0xee
```

### Any steps to reproduce the behavior?
Use Mattermost WebAPI hook instead of account info.

#### (be sure to exclude or anonymize private data (tokens/passwords))
```
[discord]
[discord.game]
Token="***"
Server="***"
ShowEmbeds=true
WebhookURL="***"


[mattermost]
[mattermost.work]
WebhookURL="***"

[[gateway]]
#REQUIRED and UNIQUE
name="gateway1"
enable=true

    [[gateway.in]]
        account="discord.game"
        channel='ID:****'

    [[gateway.out]]
        account="mattermost.work"
        channel="off-topic"

```

**Comment by 42wim** [comment]
You only want to do 1 direction relay? From discord to mattermost?

**Comment by ionbasa** [comment]
At the moment yes. I only need the incoming webhook from Mattermost? It works if I use username/password etc. I'd just rather not for this application since it's stored in plain text in the configuration file (main reason).

**Comment by 42wim** [comment]
Ok could be a regression, I guess it works with v0.15.0? Could you try?

**Comment by ionbasa** [comment]
Tried with version 0.15.0
I get the following in log:
`INFO[2017-07-12T01:26:05Z] Starting bridge: mattermost.work
INFO[2017-07-12T01:26:05Z] Connecting webhooks                           module=mattermost
`
It's referring to Mattermost. I tried with both username/password and Webhook, both give the previous error.

**Comment by 42wim** [comment]
@ionbasa this is (should be) fixed now in v0.16.1
